challenges/primes.js

```diff
@@ -10,3 +10,21 @@
 */
 
 // YOUR CODE HERE
+
+var p =[];
+function primes(max) {
+     for (var i = 2; i < max ; i++) {
+	  if (isPrime(i) === true){
+	  	p.push(i);
+	  }
+
+}
+
+function isPrime(num) {
+	for (var i = 2; i < num ; i++) {
+	  if (num % i === 0){
+	  	return false
+	  }
+	  };
```

Comment:
Watch your indentation here; it'll be easier to debug your code when it's easier to read.